packages/cdktf-cli/test/get/util.ts

```diff
@@ -28,4 +28,29 @@ export function expectImportMatchSnapshot(constraint: TerraformDependencyConstra
       expect(manifest).toMatchSnapshot();
     });
   });
+}
+
+export function expectModuleToMatchSnapshot(testName: string, testCategory: string, fixtureName: string) {
+  test(testName, async () => {
```

Comment:
I really like pulling this out into a separate function!
I'm a bit unsure whether I'd pull the `test()` function into it as well. I like the upside of being able to add other `test()` statements later on to all of those tests at once.
However at first I'd have expected a `test()` in the test file itself and was confused to find `expectModuleToMatchSnapshot()` as it looks very similar to the other matchers from jest / `expect`. Maybe some other naming using `createTest` or another verb would have helped me.
Just a comment as it came to mind, I don't have hard feelings on this but I thought I'd rather share than not 😅

Reply:
I started to make it as just a function that would be passed into `test()`, but then noticed the existing method and decided to follow suit.

Having it create the test does seem reasonable since it includes an `expect()` call, but renaming might be a good idea. Perhaps `testThatModuleMatchesSnapshot`. 🤷

Reply:
Ah, I didn't notice there already being a similar named function. Let's just keep it like it is then. Sorry for the noise 💬😅